extra/statistics-generator

```diff
@@ -63,7 +63,7 @@ collect_changes() {
     else
         # Sorting is important because we need "*-enterprise" repositories to
         # come after their Open Source counterparts (see below).
-        REPO_LIST="$("$RELEASE_TOOL" --list git --only-backend | sort)"
+        REPO_LIST="$("$RELEASE_TOOL" --list git --in-integration-version $1 --only-backend | sort)"
         REPO_LIST="$REPO_LIST integration"
```

Comment:
Question:
In newer branches `integration` will be present twice now, is this a problem?

Reply:
`release_tool` doesn't list itself (integration), so this is correct, actually. But you're right that this is a gotcha, I'm not sure if I've taken this into account every time I've written a piece of code which loops over this.

Reply:
I now came across this again, and let me make a small modification: It doesn't list itself **when using `--only-backend`**. We can debate whether this is correct I suppose, but I'm not going to change it now. If we do decide to change it later, this argument is pretty easy to search for and change everywhere.